### PR TITLE
Use Google Analytics for React Router

### PR DESCRIPTION
**Pull request by ahastudio**
Send pageview hit to Google Analytics when router link is clicked.

Send `open-menu` event to Google Analytics when user opens menu.

See also:
- [Google Analytics for Web (gtag.js) > Measure pageviews](https://developers.google.com/analytics/devguides/collection/gtagjs/pages)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -22,7 +22,6 @@
         }
 
         gtag('js', new Date());
-        gtag('config', 'UA-153178350-1');
```
Comment (ahastudio) [inline index.html]: `google-analytics.js`로 이동.

File: src/components/HeaderV3.jsx [hunk 2]
```diff
@@ -75,14 +75,19 @@ function Logo() {
 function OpenMenuButton({ active }) {
   const dispatch = useDispatch();
 
+  const handleClick = () => {
+    dispatch(openMenu());
+    window.postMessage({ type: 'open-menu' }, '*');
```
Comment (ahastudio) [inline src/components/HeaderV3.jsx]: 메뉴 열기 이벤트도 기록하게 했는데 깜빡했군요. :D

**Comment by moonkii** [comment]
icansee

**Comment by canary4651** [comment]
image

**Pull request merged**